src/components/preview/README.md

```diff
@@ -0,0 +1,27 @@
+# A Preview widget.
+
+### == How to use a Preview in a component:
+
+* In your file:
+```javascript
+   import Preview from 'carbon/lib/components/preview'
+```
+
+* To render the Preview:
+```
+  <Detail>
+   <Preview>
+     { this.props.children }
+   </Preview>
+  </Detail>
+```
+
+* The 'isVisible' boolean prop determines whether or not to render the text and children. 
+
+| Name              | Required       | Type           | Default       | Description           |
+| ----------------- |  ------------- |  ------------- | ------------- | --------------------- |
+| `isVisible`     | `false`        | `Boolean`       |  'false'      | toggle tooltip hide/show  |
+| `children`     | `false`        | `Node`       |  'null'      |   toggle whether a placeholder for content is shown |
```

Comment:
Formatting seems a bit off.

Plus I'm not sure about the descriptions either:

- The `Preview` component doesn't seem to have an `isVisible` prop. Also I'm not sure why this mentions 'tooltip' - is this a typo?
- `children` says it can toggle whether placeholder content is shown. Isn't `children` the placeholder content itself? Plus you state above that `isVisible` determines whether to render the text and children.

Reply:
👍